packages/fxa-auth-server/lib/l10n/server.ftl

```diff
@@ -9,8 +9,20 @@ session-verify-send-push-body-2 = Click here to confirm it’s you
 # $code  - 6 digit code used to verify phone ownership when registering a recovery phone
 recovery-phone-setup-sms-body = { $code } is your { -brand-mozilla } verification code. Expires in 5 minutes.
 
+# Shorter message sent by SMS with limited character length, please test translation with the messaging segment calculator
+# https://twiliodeved.github.io/message-segment-calculator/
+# Messages should be limited to one segment
+# $code  - 6 digit code used to verify phone ownership when registering a recovery phone
+recovery-phone-setup-sms-short-body = Mozilla verification code: ${code}
```

Comment:
This should use the brand term for Mozilla. Additionally - placeholders should be formatted as `{ $variable }`